pinot-core/src/main/java/org/apache/pinot/core/common/NullBitmapUtils.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.common;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import org.roaringbitmap.RoaringBitmap;
+
+
+public final class NullBitmapUtils {
+  private NullBitmapUtils() {
+  }
+
+  public static void setNullRowIds(RoaringBitmap nullBitmap, ByteArrayOutputStream fixedSizeByteArrayOutputStream,
+      ByteArrayOutputStream variableSizeDataByteArrayOutputStream)
+      throws IOException {
+    if (nullBitmap != null) {
```

Comment:
Since we didn't store the `colId` of the null bitmap, we have to write all bitmaps. When the bitmap is null or empty, we can write the offset and 0